youtube/watch.py

```diff
@@ -195,83 +198,68 @@ def lang_eq(lang1, lang2):
         return False
     return lang1[0:2] == lang2[0:2]
 
-def equiv_lang_in(lang, sequence):
-    '''Extracts a language in sequence which is equivalent to lang.
-    e.g. if lang is en, extracts en-GB from sequence.
-    Necessary because if only a specific variant like en-GB is available, can't ask Youtube for simply en. Need to get the available variant.'''
-    lang = lang[0:2]
-    for l in sequence:
-        if l[0:2] == lang:
-            return l
-    return None
-
 def get_subtitle_sources(info):
     '''Returns these sources, ordered from least to most intelligible:
     native_video_lang (Automatic)
     foreign_langs (Manual)
+
     native_video_lang (Automatic) -> pref_lang
     foreign_langs (Manual) -> pref_lang
     native_video_lang (Manual) -> pref_lang
+
     pref_lang (Automatic)
     pref_lang (Manual)'''
     sources = []
     if not yt_data_extract.captions_available(info):
         return []
-    pref_lang = settings.subtitles_language
+
+    user_langs = settings.subtitles_language.split(',')
     native_video_lang = None
     if info['automatic_caption_languages']:
         native_video_lang = info['automatic_caption_languages'][0]
 
-    highest_fidelity_is_manual = False
+    kManual, kNativeVideoLang, kUserSpeaksIt = 1, 2, 4
```

Comment:
Shouldn't this be `kNativeVideoLang, kManual, kUserSpeaksIt = 1, 2, 4`?

Manually created subtitles in a different language than the video are always going to be higher quality than automatically generated ones. Comparing the current scores to the comment at the top of the code with the ideal ordering:
```010    native_video_lang (Automatic)
001    foreign_langs (Manual)

110    native_video_lang (Automatic) -> pref_lang
101    foreign_langs (Manual) -> pref_lang
111    native_video_lang (Manual) -> pref_lang

110   pref_lang (Automatic)
111   pref_lang (Manual) (101 if pref_lang != native_video_lang)'''
```
Swapping the last two bits will fix it

Additionally, these constants should be changed to `NATIVE_VIDEO_LANG, MANUAL`, etc. to match the convention in the rest of the code that constants are indicated with uppercase snakecase; we dont use hungarian notation in the code.